src/leaflike/layout.clj

```diff
@@ -14,15 +14,21 @@
 (defn application
   [title content & {:keys [username error-msg success-msg]}]
   (html5 [:head
+          [:meta {:charset "utf-8"}]
           [:title title]
           (include-css "//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css")
           (include-css "/css/styles.css")
           (include-css "/css/select2.min.css")
           (include-js "/js/jquery.min.js")
           (include-js "/js/select2.min.js")
-          [:body
+          [:link {:rel "shortcut icon"
+                  :href "/favicon.ico?"
```

Comment:
Firefox doesn't load the favicon unless we have `?` at the end of filepath. We are not sure why.